main_logic/core.py

```
@@ -2739,20 +2740,29 @@ async def _process_stream_data_internal(self, message: dict):
                 try:
                     # 使用统一的屏幕分享工具处理数据（只验证，不缩放）
                     image_b64 = await process_screen_data(data)
-                    
+
                     if image_b64:
+                        # 叠加 Avatar 文字注解（如果有位置元数据）
```

Ruff (0.15.9)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)